README.md

```diff
@@ -84,19 +377,12 @@ rake osc:build
 ```
```

Comment:
Note that `rake osc:build` only builds libyui.spec. I am not sure how to indicate to OBS to consider the rest of spec files. OBS does it automatically thanks to the *_multibuild* file, but no idea about osc.

Reply:
I don't know about that. I left this section unchanged.